src/bootstrap/lib.rs

```diff
@@ -1570,11 +1561,9 @@ impl Build {
             // Only copy file or directory if the filter function returns true
             if filter(&relative) {
                 if t!(f.file_type()).is_dir() {
-                    let _ = fs::remove_dir_all(&dst);
                     self.create_dir(&dst);
                     self.recurse_(&path, &dst, &relative, filter);
                 } else {
-                    let _ = fs::remove_file(&dst);
```

Comment:
Can you say more about removing these lines? What is replacing them / why is it safe to do that?

Reply:
I've described why this line can be removed in commit's title:
`dont call remove_file in cp_filtered, as self.copy will call copy_internal which will call remove_file itself`

Hm, looks like i mean `recurse_` function, not `cp_filtered`

Reply:
Comment about `remove_dir_all`: https://github.com/rust-lang/rust/pull/111734#issuecomment-1556148879